src/OpenLoco/Windows/ToolbarTop.cpp

```diff
@@ -811,8 +807,12 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Game
             window->widgets[Common::Widx::view_menu].left = baseWidget.left + 14 + (baseWidget.width() * 3);
         }
 
-        if (last_port_option == 0 && addr<0x00525FAC, int8_t>() != -1 && addr<0x00525FAD, int8_t>() == -1)
+        if (last_port_option == 0
```

Comment:
this `if` statement seemed strange to me, not sure why it's needed/why it does what it does